lodash-plugin/src/test/java/com/github/underscore/lodash/LodashTest.java

```diff
@@ -589,84 +589,84 @@ public void chain() {
+            public boolean test(String str) { return true; } });
         $.chain(new String[] {""}).reduce(new FunctionAccum<String, String>() {
             public String apply(String accum, String str) { return null; } }, "");
         $.chain(new String[] {""}).reduceRight(new FunctionAccum<String, String>() {
             public String apply(String accum, String str) { return null; } }, "");
         $.chain(new String[] {""}).find(new Predicate<String>() {
-            public Boolean apply(String str) { return true; } });
+            public boolean test(String str) { return true; } });
         $.chain(new String[] {""}).findLast(new Predicate<String>() {
-            public Boolean apply(String str) { return true; } });
+            public boolean test(String str) { return true; } });
         $.chain(new Integer[] {0}).max();
-        $.chain(new Integer[] {0}).max(new Function1<Integer, Integer>() {
+        $.chain(new Integer[] {0}).max(new Function<Integer, Integer>() {
             public Integer apply(Integer value) { return value; } });
         $.chain(new Integer[] {0}).min();
-        $.chain(new Integer[] {0}).min(new Function1<Integer, Integer>() {
+        $.chain(new Integer[] {0}).min(new Function<Integer, Integer>() {
             public Integer apply(Integer value) { return value; } });
         $.chain(new Integer[] {0}).sort();
         $.chain(new Integer[] {0}).sortWith(new Comparator<Integer>() {
             public int compare(Integer value1, Integer value2) { return value1; } });
-        $.chain(new Integer[] {0}).sortBy(new Function1<Integer, Integer>() {
+        $.chain(new Integer[] {0}).sortBy(new Function<Integer, Integer>() {
             public Integer apply(Integer value) { return value; } });
         $.chain(new LinkedHashMap<Integer, Integer>().entrySet()).sortBy("");
-        $.chain(new Integer[] {0}).groupBy(new Function1<Integer, Integer>() {
+        $.chain(new Integer[] {0}).groupBy(new Function<Integer, Integer>() {
             public Integer apply(Integer value) { return value; } });
         $.chain(new Integer[] {0}).indexBy("");
-        $.chain(new Integer[] {0}).countBy(new Function1<Integer, Integer>() {
+        $.chain(new Integer[] {0}).countBy(new Function<Integer, Integer>() {
             public Integer apply(Integer value) { return value; } });
         $.chain(new Integer[] {0}).shuffle();
         $.chain(new Integer[] {0}).sample();
         $.chain(new Integer[] {0}).sample(1);
         $.chain(new int[] {0}).value();
-        $.chain(new String[] {""}).tap(new Block<String>() {
-            public void apply(String str) {
+        $.chain(new String[] {""}).tap(new Consumer<String>() {
+            public void accept(String str) {
             } });
-        $.chain(new String[] {""}).forEach(new Block<String>() {
-            public void apply(String str) {
+        $.chain(new String[] {""}).forEach(new Consumer<String>() {
```

Comment:
\[FindBugs\] INFO: RV: Return value of $$Chain\.forEach\(Consumer\) ignored, is this OK in com\.github\.underscore\.lodash\.LodashTest\.chain\(\)